gradle/libs.versions.toml

```diff
@@ -20,7 +20,8 @@ opentelemetryContribAlpha = "1.44.0-alpha"
 # updated from upstream agent with .ci/update-upstream.sh
 # While the semconv stable artifact is provided transitively by the agent, we still have to explicitly
 # reference the "incubating" version explicitly
-opentelemetrySemconv = "1.30.0-rc.1"
+opentelemetrySemconv = "1.30.0"
+opentelemetrySemconvAlpha = "1.30.0-alpha"
```

Comment:
We'll also have to update the automation for those to not go out of sync, don't we?

Reply:
Very good point, I'll implement that.